spec/module.dd

```diff
@@ -318,8 +318,8 @@ void test()
 $(H2 $(LNAME2 public_imports, Public Imports))
 
 $(P By default, imports are $(I private). This means that if module A imports
-module B, and module B imports module C, then names from C are visible only from
-B and not from A.)
+module B, and module B imports module C, then names inside C are visible only inside
+B and not inside A.)
```

Comment:
I think this is good as is.

Reply:
I think that 'inside' is clearer. After all 'from' doesn't tell us where they originate (could be the imported module or one of the modules it publically imports).
Again, I'll leave you to revert (or presumably just reject).